### ISSUE TITLE
Data file sizes

### ISSUE THREAD

**Issue by paulmartini** [desihub/desiperf#53]
Hi @parfa30 -- would you take a look the changes in my branch to accommodate a .fits.gz file instead of the .csv files? I'm getting errors (so please don't merge this), and would appreciate it if you see something obviously wrong with what I've done. Thanks!

**Comment by parfa30** [comment]
Whoo, that was tricky! I'm pretty sure I've figured it out though. When pandas tables are are saved in fits files, it seems to turn all strings into byte objects. This was tripping us up with 'obstype' and 'program'. So, for example, 'SCIENCE' --> 'B"SCIENCE"'. We will have to fix that on one side or another - either when we read the file in or when saving it originally.

**Comment by paulmartini** [comment]
Good catch! I think you mean that when pandas converts a table into a pandas dataframe, it converts strings to byte objects. The funny thing about this is that I do nfot get this error when I execute the exact same commands in a notebook -- just when I run the app.

**Comment by parfa30** [comment]
Yeah, not sure when it happens. When I was saving the files as csv it kept them as strings. I'm sure there's some clever way to check for it and correct it when we read in the files, but I've had trouble with this before.

**Comment by parfa30** [comment]
`fp_df['obstype'].str.decode("utf-8")`

**Comment by parfa30** [comment]
Hey Paul, I"m going to push these changes. What is the code that you used to save the csv files into fits.gz?
